core/src/br/cefetmg/games/screens/Overworld.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package br.cefetmg.games.screens;
+
+import br.cefetmg.games.minigames.util.GameOption;
+import br.cefetmg.games.minigames.util.GameType;
+import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Input;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.Sprite;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.utils.Timer;
+/**
+ *
+ * @author thais
+ */
+
+
+public class Overworld extends BaseScreen{
+
+    public static final boolean DEV_MODE = true;
+    
+    private final Texture background;
+    //targetTexture;
+    private Stage stage;
+    private Object screen;
+    
+    public Overworld(Game game, BaseScreen previous) {
+        super(game, previous);  
+        
+        //TODO assets.get
+        super.assets.load("overworld/map.png", Texture.class);
+        background = new Texture("overworld/map.png");
+        
+   
+    }
+
+    @Override
+    public void appear() {
+
+    }
+
+    @Override
+    public void cleanUp() {
+    }
+
+    @Override
+    public void handleInput() {
+        if(Gdx.input.isButtonPressed(Input.Buttons.LEFT)){
+
+            if (Gdx.input.getX() > 0 && Gdx.input.getX() < 260
```

Comment:
Há formas bem melhores disponibilizadas pela LibGDX para identificar se uma coordenada está dentro de um retângulo. Além disso, usar números mágicos (260, 420 etc.) é uma prática muito ruim de programação porque eles não têm nenhuma semântica.

Veja aqui um exemplo: http://stackoverflow.com/questions/25460029/is-this-the-right-way-to-detect-a-touch-on-a-rectangle-in-libgdx-does-not-seem (veja a resposta marcada como certa)